templates/base.html

```diff
@@ -0,0 +1,81 @@
+    <head>
+        <meta charset="utf-8">
+        <meta name="viewport" content="width=device-width, initial-scale=1">
+
+        <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.0-beta1/dist/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-giJF6kkoqNQ00vy+HMDP7azOuL0xtbfIcaT9wjKHr8RbDVddVHyTfAAsrekwKmP1" crossorigin="anonymous">
+        <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bootstrap-icons@1.3.0/font/bootstrap-icons.css">
+    <!--    <link rel="stylesheet" href="/static/css/base.css">-->
+
+        <title> replace me </title>
+    </head>
+    <body class="d-flex flex-column h-100 min-vh-100">
+        <nav class="navbar navbar-expand-lg navbar-dark bg-dark">
+            <div class="container-fluid">
+                <a class="navbar-brand" href="/" >בית</a>
+                <button class="navbar-toggler" type="button" data-bs-toggle="collapse" data-bs-target="#navbarNav" aria-controls="navbarNav" aria-expanded="false" aria-label="Toggle navigation">
+                    <span class="navbar-toggler-icon"></span>
+                </button>
+                <div class="collapse navbar-collapse" id="navbarNav">
+                    <ul class="navbar-nav">
+                        <li class="nav-item">
+                            <a class="nav-link active" aria-current="page" href="/about/">עלינו</a>
+                        </li>
+                        <li class="nav-item">
+                            <a class="nav-link active" href="/contact/">יצירת קשר</a>
+                        </li>
+                            {%if not request.user.is_anonymous%}
+                                <span class="navbar-text px-2">
+                                    ברוך הבא, {{request.user}}
+                                </span>
+                            {%endif%}
+                        <li class="nav-item">
+                            {%if request.user.is_anonymous%}
+                                <a class="nav-link" href="/login/">כניסה</a>
+                            {%else%}
+
+                                <a class="nav-link" href="/logout/">יציאה</a>
+                            {%endif%}
+                        </li>
+                    </ul>
+                </div>
+            </div>
+        </nav>
+        <main class="flex-fill">
+            <div class="container-fluid h-100">
+                <div class="row text-center justify-content-center">
+                    <h1 class="mb-5 mt-5">{% block title %} {%endblock%}</h1>
+                    <div class="col-xxl-10 col-xl-12">
+                        {% block content %}
+                        {% endblock %}
+                    </div>
+                </div>
+            </div>
+        </main>
+        <div class="footer container mb-5 mt-3">
+            <div class="row">
+                <div class="col text-end">
+                    <button class="btn btn-outline-dark bi-chevron-right" onclick="goBack()" >חזרה</button>

```

Comment:
I mean... You can go back on the home page if you want to return to wherever you were before that

Reply:
But if you don't have history? Hide it

Reply:
done that (although my solution is a bit junky) If there is a way to get the DOM info from the django template thing then it'll be better since I could just not create the button...